doc/source/whatsnew/v0.18.1.rst

```diff
@@ -77,9 +77,52 @@ Previously you would have to do this to get a rolling window mean per-group:
    df = pd.DataFrame({"A": [1] * 20 + [2] * 12 + [3] * 8, "B": np.arange(40)})
    df
```

Comment:
Change this DataFrame to make the below output be of reasonable size: `df = pd.DataFrame({"A": [1] * 10 + [2] * 6 + [3] * 4, "B": np.arange(20)})`

Reply:
Can you explain the motivation here? In general, I don't think we should be rewriting old release notes unless there is an issue we're fixing.